aws/resource_aws_secretsmanager_secret.go

```diff
@@ -154,6 +175,24 @@ func resourceAwsSecretsManagerSecretRead(d *schema.ResourceData, meta interface{
 	d.Set("description", output.Description)
 	d.Set("kms_key_id", output.KmsKeyId)
 	d.Set("name", output.Name)
+
+	pIn := &secretsmanager.GetResourcePolicyInput{
+		SecretId: aws.String(d.Id()),
+	}
+	log.Printf("[DEBUG] Reading Secrets Manager Secret policy: %s", pIn)
+	pOut, err := conn.GetResourcePolicy(pIn)
+	if err != nil {
+		return fmt.Errorf("error reading Secrets Manager Secret policy: %s", err)
+	}
+
+	if pOut.ResourcePolicy != nil {
+		policy, err := structure.NormalizeJsonString(*pOut.ResourcePolicy)
```

Comment:
To prevent potential panics, we should prefer to use the SDK provided function for dereferencing values (`aws.StringValue(pOut.ResourcePolicy)`) instead of directly dereferencing via `*`.